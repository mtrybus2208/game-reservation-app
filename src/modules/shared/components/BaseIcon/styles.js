import styled from 'styled-components';

export const BaseIcon = styled.img`
  display: flex;  
  margin: 0;
  padding: 0;
  width: ${({ size }) => size};
`;
