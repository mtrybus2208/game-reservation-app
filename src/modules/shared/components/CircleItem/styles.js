import styled, { css } from 'styled-components';
import { rgba } from 'polished';

export const CircleItem = styled.div`
    position: relative;
    border: solid 5px #23272d;
    box-shadow:
        0px 17px 24px 0 rgba(0, 0, 0, 0.33),
        inset 0px 2px 3px 0 rgba(255, 255, 255, 0.1);
    border: none;
    background-color: ${({ active }) => active ? '#bd790b' : '#23272d'};
    flex: 1 1 100%;
    border-radius: 100%;
    ${props => props.hoverable && css`
      &:hover {
        background: ${({ active }) => active ? '#bd790b' : rgba('#bd790b', 0.3)};
      }
    `}
`;

export const Body = styled.div`
    width: 85%;
    height: 85%;
    overflow: hidden;
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%,-50%);
    box-shadow: -1px 1px 30px -3px rgba(0,0,0,0.21);
    border: none;
    background-color: #191b21;
    border-radius: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
`;
