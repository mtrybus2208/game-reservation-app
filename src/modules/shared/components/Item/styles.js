import styled from 'styled-components';

export const Item = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  flex: 1;
`;
