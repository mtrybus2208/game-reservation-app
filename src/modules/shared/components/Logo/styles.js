import styled from 'styled-components';

export const Logo = styled.div`
    display: flex;
    flex-direction: row;
    height: 35px;
`;

export const Image = styled.img`
    height: auto;
    box-sizing: content-box;
    width: 26px;
    padding: 0 21px;
    @media ${({ theme }) => theme.media.laptop} {
        width: 30px;
        padding: 0 16px 0 30px;
        border-right: 2px solid #6d6e6f;
    }
`;

export const Title = styled.div`
    margin-top: 3px;
    @media ${({ theme }) => theme.media.laptop} {
        margin-top: 0;
        margin-left: 18px;
    }
`;

export const Header = styled.div`
    font-family: Roboto;
    font-size: 13px;
    font-weight: normal;
    line-height: 1.19;
    letter-spacing: 2.4px;
    text-align: left;
    text-transform: uppercase;
    color: #6d6e6f;
    @media ${({ theme }) => theme.media.laptop} {
        font-size: 17px;
        line-height: 16px;
        letter-spacing: 3.15px;
    }
`;

export const HeaderHighlighted = styled.span`
    color: #8f6618;
`;

export const Description = styled.div`
    font-family: Roboto;
    font-size: 10px;
    font-weight: 300;
    line-height: 1.94;
    letter-spacing: .2px;
    text-align: left;
    color: #8f6618;
    @media ${({ theme }) => theme.media.laptop} {
        margin-top: 5px;
        font-size: 11px;
        line-height: 1.45;
        letter-spacing: .3px;
    }
`;