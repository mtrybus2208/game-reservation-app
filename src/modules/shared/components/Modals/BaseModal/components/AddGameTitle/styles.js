import styled from 'styled-components';

export const Bold = styled.span`
  color: ${props => props.theme.accent};
`;
