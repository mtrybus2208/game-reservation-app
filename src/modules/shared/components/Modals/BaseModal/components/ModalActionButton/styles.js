import styled from 'styled-components';
import { rgba } from 'polished';

export const ModalActionButton = styled.button`
  cursor: pointer;
  display: flex;
  justify-content: center;
  align-items: center;
  background: none;
  border: none;
  padding: 15px;
  flex-grow: 1;
  height: 100%;
  outline-color: ${props => rgba(props.theme.accent, .2)};
`;
