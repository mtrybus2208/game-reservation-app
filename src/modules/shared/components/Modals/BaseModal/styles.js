import styled from 'styled-components';

export const Modal = styled.div`
  background: #111315;
  min-height: 207px;
  display: flex;
  max-width: 400px;
  min-width: 350px;
  width: 100%;
  color: #111315;
  flex-direction: column;
  box-shadow: 0px 17px 24px 0 rgba(0, 0, 0, .33), inset 0px 0px 1px 0 rgba(255, 255, 255, .1);
  background-image: linear-gradient(to top,rgba(24, 25, 27, 0.1),rgba(32, 36, 41, .1));
`;

export const Header = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 15px;
  text-align: center;
  box-shadow: 0px 17px 24px 0 rgba(0, 0, 0, .33), inset 0px 0px 1px 0 rgba(255, 255, 255, .1);
  flex: 0; 
  padding: 25px;
`;

export const Title = styled.div`
  color: #716b6b;
  text-transform: uppercase;
  font-size: 14px;
`;

export const Body = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 15px;
  flex: 1 1 100%;
`;
 