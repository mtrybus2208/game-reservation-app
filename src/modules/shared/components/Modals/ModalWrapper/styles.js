import styled, { css } from 'styled-components';
import { rgba } from 'polished';

export const ModalWrapper = styled.div`
  position: fixed;
  left: 0;
  right: 0;
  top: 0;
  bottom: 0;
  display: flex;
  justify-content: center;
  align-items: center;
  z-index: 10;
  background: rgba(0, 0, 0, .6);
`;
