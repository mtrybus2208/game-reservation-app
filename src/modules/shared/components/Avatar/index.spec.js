import React from 'react';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import Avatar from './index';
import * as S from './styles';

describe('<Avatar />', () => {
  it('renders without crashing', () => {
    shallow(<Avatar />);
  });

  it('should matches the snapshot', () => {
    const avatar = shallow(<Avatar />);

    expect(toJson(avatar)).toMatchSnapshot();
  });

  it('should render without crashing given the props', () => {
    const props = {
      path: 'test',
      size: 'md',
      rounded: true,
    };
    const avatar = shallow((<Avatar {...props} />));

    expect(toJson(avatar)).toMatchSnapshot();
  });

  it('should set the path prop as the `src` prop on the S.Image component', () => {
    const props = {
      path: 'test',
      size: 'md',
      rounded: true,
    };
    const wrapper = shallow(<Avatar {...props} />);
    const SImage = wrapper.find(S.Image);

    expect(SImage.props().src).toBe(props.path);
  });
});
