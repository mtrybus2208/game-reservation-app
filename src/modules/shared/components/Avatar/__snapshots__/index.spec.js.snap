// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> should matches the snapshot 1`] = `
<StyledAvatar
  rounded={true}
>
  <styled.div>
    <styled.img
      src="https://res.cloudinary.com/duo6ruqkc/image/upload/v1550848181/child_rtmjgd.png"
    />
  </styled.div>
</StyledAvatar>
`;

exports[`<Avatar /> should render without crashing given the props 1`] = `
<StyledAvatar
  rounded={true}
  size={100}
>
  <styled.div>
    <styled.img
      src="test"
    />
  </styled.div>
</StyledAvatar>
`;
